farcy/__init__.py

```diff
@@ -100,7 +101,7 @@ def __init__(self, config):
                                  .format(self.config.repository))
         # Keep track of open pull requests
         self.open_prs = {}
-        for pr in self.repo.pull_requests(state='all'):
+        for pr in self.repo.pull_requests(state='open'):
             if pr.state == 'open':
```

Comment:
Yes. When processing from a given start ID a pull request may be opened and later closed. We don't want to process pull requests that are already closed.